ibis/expr/types/relations.py

```diff
@@ -1191,6 +1191,80 @@ def distinct(
             return res.select(self.columns)
         return res
 
+    def sample(
+        self,
+        fraction: float,
+        *,
+        method: Literal["row", "block"] = "row",
+        seed: int | None = None,
+    ) -> Table:
+        """Sample a fraction of rows from a table.
```

Comment:
Is the caveat of not returning the same number of rows for the same value of `fraction` worth a `::: {.callout-warning}` section?

Reply:
Good idea. Went with a `.callout-note` since that's used in other places in the same file for a similar purpose.